velox/vector/tests/VectorTest.cpp

```diff
@@ -1716,3 +1717,61 @@ TEST_F(VectorTest, selectiveLoadingOfLazyDictionaryNested) {
   dict->loadedVector();
   ASSERT_EQ(loaderPtr->rowCount(), 1 + size / 4);
 }
+
+TEST_F(VectorTest, dictionaryResize) {
+  auto vectorMaker = std::make_unique<test::VectorMaker>(pool_.get());
+  vector_size_t size = 10;
+  std::vector<int64_t> elements{0, 1, 2, 3, 4};
+  auto makeIndicesFn = [&]() {
+    return makeIndices(size, [&](auto row) { return row % elements.size(); });
+  };
+
+  auto indices = makeIndicesFn();
+  auto flatVector = vectorMaker->flatVector(elements);
+
+  // Create a simple dictionary.
+  auto dict = BaseVector::wrapInDictionary(
```

Comment:
VectorTestBase has a helper method wrapInDictionary that doesn't require passing in nulls.